pkg/ebpf/tracer.go

```diff
@@ -20,11 +20,17 @@ import (
 )
 
 var (
-	probeExpvar *expvar.Map
+	conntrackExpvar *expvar.Map
```

Comment:
nit: can we have a map instead of this ?

```suggestion
	expvarEndpoints map[string]*expvar.Map
```